hexa/workspaces/models.py

```diff
@@ -152,8 +152,7 @@ def db_port(self):
 
     @property
     def db_url(self):
-        host = get_db_server_credentials()["host"]
-        return f"postgresql://{self.db_name}:{self.db_password}@{host}:{self.db_port}/{self.db_name}"
+        return f"{self.slug}.{settings.WORKSPACES_DATABASE_PROXY_HOST}"
```

Comment:
@cheikhgwane only the IP should be replaced by xyz.explore.openhexa.org.

By the way, `f"{self.slug}.{..."` could be replaced by `self.db_host` ;)

Reply:
Ah yes, my bad.

Reply:
Something like this ?
```python
f"postgresql://{self.db_name}:{self.db_password}@{settings.WORKSPACES_DATABASE_PROXY_HOST}:{self.db_port}/{self.db_name}"
```